minard/views.py

```diff
@@ -546,6 +547,27 @@ def l2():
         return redirect(url_for('l2',step=step,height=height,_external=True))
     return render_template('l2.html',step=step,height=height)
 
+@app.route('/burst')
+def burst():
```

Comment:
Here I think you should add something specifying how many to load and at what offset. Basically, the issue is that as you get more and more bursts, eventually this page will take a really long time to load and will be *very* big. What you can do is something called "pagination" where you first load the first N results and then add, "Next" and "Back" buttons to the page.

Reply:
See https://github.com/snoplus/minard/blob/master/minard/orca.py and https://github.com/snoplus/minard/blob/master/minard/HLDQTools.py#L5 for examples on using the database and https://github.com/snoplus/minard/blob/master/minard/templates/orca_session_logs.html and https://github.com/snoplus/minard/blob/master/minard/templates/physicsdq.html for examples on how to do the template.

Reply:
I have addressed this in pretty much the same way but still using the couchdb access